guidelines/REUSE.md

```diff
@@ -1,85 +1,100 @@
 # Reuse Philosophy
+
 We encourage reuse and patterns through references.
 
 ## What is reusable
+
 The following types are reusable, as defined by the spec:
 
-* Operations
 * Parameters
-* Responses
 * Models (or Schema Objects in general)
+* Responses
+* Operations (_Operations can only be remote references_)
 
 ## Reuse strategy
-When reusing components in an API design, a pointer is created from the definition to target design.  The references are maintained in the structure, and can be updated by modifying the source definitions.  This is different from a "copy on design" approach where references are injected into the design itself.
 
-The reuse technique is transparent between JSON or YAML and is lossless when converting between the two.
+When reusing components in an API design, a reference is created from the definition to target entity. The references are maintained in the structure, and can be updated by modifying the source definitions. This is different from a "copy on design" approach where references are injected into the design itself.
```

Comment:
I really hated this paragraph, and was out of energy for it. Thanks for the addition, I'll reword a bit for consistency of voice and get it in there.